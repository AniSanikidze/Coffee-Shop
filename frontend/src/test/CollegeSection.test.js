import * as React from "react";
import * as ReactDOM from "react-dom";
import CollegeSection from "../components/college/CollegeSection";
import * as enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16'
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom'
import { configure } from 'enzyme';
import { UserContext } from "../UserContext";

configure({ adapter: new Adapter() });

describe('testing CollegeSection', () => {

    it('should render', () => {
        const CollegeSectionComponent = renderer.create(<UserContext.Provider value={false}><Router><CollegeSection /></Router></UserContext.Provider>).toJSON();
        expect(CollegeSectionComponent).toMatchSnapshot();
    })

})