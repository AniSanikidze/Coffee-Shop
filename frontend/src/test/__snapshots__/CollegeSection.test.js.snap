// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing CollegeSection should render 1`] = `
<div
  className="college-section"
>
  <div
    className="container"
    data-aos="fade-up"
  >
    <div
      className="row home__hero-row"
      style={
        Object {
          "display": "flex",
          "flexDirection": "row",
        }
      }
    >
      <div
        className="col"
      >
        <div
          className="text-wrapper"
        >
          <div
            className="top-line"
          >
            For Prague College Students
            <img
              alt="pc logo"
              className="prague-college-logo"
              src="images/Home/PC.png"
            />
          </div>
          <h1
            className="heading"
          >
            Find the best food places near the Prague College
          </h1>
          <p
            className="description"
          >
            We would like to help Prague College students to find restaurants for their lunch-breaks as soon as possible. Click on Get Started and find a suitable restaurant based on your own preferences.
          </p>
          <a
            href="/restaurants"
            onClick={[Function]}
          >
            <button
              className="btn btn--search btn--large"
            >
              Get Started
            </button>
          </a>
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="img-wrapper"
        >
          <img
            alt="Student"
            className="student-img"
            src="images/Home/lnch.jpg"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
